### PR TITLE
Weather Forecast App

### PR DESCRIPTION
**Pull request by acushlakoncept**
In this PR I created a weather forecast app using the open map weather API.
I got comfortable consuming the API using Async/Await to work with promises using the `fetch` function.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "style-loader": "^1.2.1",
+    "webpack": "^4.44.2",
+    "webpack-cli": "^3.3.12"
+  },
+  "dependencies": {
+    "bootstrap": "^4.5.2",
+    "jquery": "^3.5.1",
+    "popper.js": "^1.16.1"
+  }
+}
```
Comment (lekegitrepo) [inline package.json:49]: - **_The build failed; as indicated in the image :point_down:_**

Screenshot from 2020-09-30 11-54-52

File: README.md [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+
+
+## Getting Started
+
+To get a local copy of the repository please run the following commands on your terminal:
+
+```bash
+$ cd <folder>
+$ git clone https://github.com/acushlakoncept/weather-app.git
+$ cd todo
```
Comment (lekegitrepo) [inline README.md]: - **_Change this to the correct name._**

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Request changes :recycle:
### Hi @acushlakoncept :clap: :clap:
#### Well done :+1: you have done a good job. However, the following :point_down: issues need to be fix be this project can be approved.

- **_Fix the build issue; when I ran the command `npm run start` it outputs error as specified in the image of the inline comment below_**

**_Kindly, fix these and submit for them another review._**

#### _Reviewed by Temitayo Adeleke._

File: README.md [hunk 3]
```diff
@@ -1,3 +1,49 @@
...
+## Live Demo
+[Live Demo Link](https://acushlakoncept.github.io/weather-app/)
+
+
+## Getting Started
+
+To get a local copy of the repository please run the following commands on your terminal:
+
+```bash
+$ cd <folder>
```
Comment (mosesogwo) [inline README.md]: Do we really need this first line of instruction?

**Review by mosesogwo — APPROVED** [review]
**Well done @acushlakoncept. The build works well now:+1:**

### Status: Approved :heavy_check_mark:

File: README.md (hunk 3 above)
Comment (acushlakoncept) [reply README.md]: removed!

**Pull request merged**